Label physics layer flyout more specifically

RayKoopa opened the pull request:
S. https://github.com/godotengine/godot/issues/14363
Quickly threw a PR together to apply further discussion on. In case the naming should be changed to something different, feel free to yell me some better titles. 💃 

I used `hint_text` to provide more specific titles for the physics layer flyout.
Those are not translated as they previously weren't either. Tell me to put a `TTR` around the `set_text` or each properties hint text.

And also please let me know if `hint_text` was a bad idea.
![image](https://user-images.githubusercontent.com/9631152/33716068-055522a0-db55-11e7-9ae3-5ce084a6b4db.png)

RayKoopa committed:
Label physics layer flyout more specifically

akien-mga commented:
I think they should be made translatable with `TTR` yeah.

Now, this change exposes the same kind of issue for the 2D/3D render parameters as there was for the physics layers, so I'm not sure it's the right solution.
Currently all render parameters will be listed as "2D/3D Render Mask" when the actual properties are light culling, range item culling, shadow item culling, occluding lighting and "normal" culling. It's maybe a tad less confusing that the physics/collisions layers/mask issue which we've never managed to properly solve, so I guess it's not too bad - just wanted to point out that it's not consistent though (for physics we show the name of the different properties which both rely on the same object, the physics layers; for rendering we should the name of the common object used by all those properties, the render mask).

groud commented:
Hum, I don't think the hint_text is a good idea, as the HINT are already very specific.
To be honest, I think the issue it intends to solve is not even an issue.

akien-mga commented:
What this popup shows is the physics layers in both case, so the current name is indeed correct, even though it may be a bit confusing due to the name "collision layers" property. It's actually "2D/3D Physics Layers used as collision layers" and "2D/3D Physics Layers used as collision mask".

RayKoopa commented:
Yeah, I understood it before aswell.
One property I changed can also be set with `set/get_layer_mask` to which I added the title "3D Render Layers" 🤣. You decide.

akien-mga commented:
> (Maybe it's still better to get rid of the dumb label altogether?)

I think that's the way to go. @reduz agreed too on IRC.
Closing this PR, but feel free to make a new one removing the confusing labels.

RayKoopa commented:
Will do as soon as I am home
